### DISCUSSION TITLE
Add Keybindings column to Set Custom Command dialog

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4067] [Ideas]
Depends how much typical users are using the “Set Custom Command” dialog?
Can close this if no one is interested.

I noticed that there are not many visual indicators of what keybindings are set to “Send to Custom Command” tools.  
-"Preferences - Keybindings" just shows generic “Send To Custom Command 1 - 9” with the keybindings
-"Set Custom Command" dialog just shows the command Labels
-The only place that shows both the names + keybindings together is the menu bar flyout but this is not a place where any setting/editing can be achieved.

The only thing I can think of is to maybe add a “Keybinding” column to the end of the “Set Custom Command” dialog?
ID / Command / Label / Keybinding

Not everyone might use Ctrl+1-9 keybindings where the keybindings are in a convenient numbered order 1-9.
This “Keybinding” column would let user know that when they are moving items up/down on the list that the actual keybinding are not moving with the actual command actions.  The keybindings linked are fixed to 1-9 order.

 I do not see anyone programming the ability to be able to actually set the keybindings from this dialog, which would update the keybindings shown in Preferences - Keybindings.
So the “Keybinding” column would be more static where it cannot be edited, but would just be an indicator of what keybindings are actually linked to those 1-9 positions.

Thank You

**Comment by b4n** [comment]
Makes sense I guess, if we can convey that it's not meant to be modified there; it can indeed be handy to see which binding is used for what and how moving items affects mapping. 

Possibly we could even allow modifying them through that dialog like if one used the preferences dialog if that's easy enough, but potentially it'd be more confusing given there's a limited number of available bindings and you couldn't set e.g. the 10th command binding.
